DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/platform_designer_standard/README.md

```diff
@@ -0,0 +1,401 @@
+# `Platform Designer Standard` Sample
+
+This example design shows how to use an FPGA IP produced with the Intel® oneAPI DPC++/C++ Compiler with the Intel® Quartus® Prime Standard Edition to target a Cyclone® V FPGA.
```

Comment:
you can modify the OG platform designer sample to remove mention of Quartus std since we show it here now.